frontend/src/components/RepositorySetupWizard.vue

```diff
@@ -69,11 +69,14 @@ const DEFAULT_FILE_PATH_TEMPLATE =
   "{{ENV_NAME}}/{{DB_NAME}}__{{VERSION}}__{{TYPE}}__{{DESCRIPTION}}.sql";
 // Default schema path template is co-locate with the corresponding db's migration files and use .(dot) to appear the first.
 const DEFAULT_SCHEMA_PATH_TEMPLATE = "{{ENV_NAME}}/.{{DB_NAME}}__LATEST.sql";
+const DEFAULT_SHEET_PATH_TEMPLATE =
+  "sheet/{{ENV_NAME}}__{{DB_NAME}}__{{NAME}}.sql";
```

Comment:
```suggestion
//  Use script/xxx instead of sheet/xxx because the db user is more familiar with the former concept as SQL script. We name sheet internally because we may expand its scope later.
const DEFAULT_SHEET_PATH_TEMPLATE =
  "sheet/{{ENV_NAME}}__{{DB_NAME}}__{{NAME}}.sql";
```